tools/querytee/proxy_metrics.go

```diff
@@ -40,9 +40,10 @@ func NewProxyMetrics(registerer prometheus.Registerer) *ProxyMetrics {
 		}, []string{"backend", "route", "result", "issuer"}),
 		missingMetrics: promauto.With(registerer).NewHistogramVec(prometheus.HistogramOpts{
 			Namespace: "cortex_querytee",
-			Name:      "missing_metrics",
+			Name:      "missing_metrics_series",
 			Help:      "Number of missing metrics (series) in a vector response.",
-		}, []string{"backend", "method", "route", "status_code", "issuer"}),
+			Buckets:   []float64{.005, .01, .025, .05, .1, .25, .5, 0.75, 1, 1.5, 2, 3, 4, 5, 10, 25, 50, 100},
```

Comment:
Hhmm, not sure why I didn't notice this before but why is this a histogram and not a counter, for example?
How do we plan to use this metric?

Reply:
OK cool, let's give it a go 👍